modules/omr-registry-localhost-mirroring.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="omr-registry-localhost-mirroring_{context}"]
+
+= Local host mirroring with Red Hat Openshift Mirror Registry
+
+This procedure introduces how to install Red Hat Quay on a local host using the `openshift-mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
+
+[NOTE]
+====
+Installing Red Hat Quay using the `openshift-mirror-registry` CLI tool makes several changes to your machine. Upon installation, an `/etc/quay-install` directory is created, which contains installation files, local storage, and the config bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
+====
+
+.Procedure
+
+. Download the `openshift-mirror-registry-offline.tar.gz` package for the latest version of OMR found on the link:https://github.com/quay/openshift-mirror-registry/releases[OMR releases page].
++
+[NOTE]
+====
+If you are using the online version, additional container images to run Red Hat Quay and Postgres will be downloaded by the installer. If you are using the offline version, it will contain all required images to run Red Hat Quay.
+====
+
+. Install Red Hat Quay on your local host with your current user account:
++
+[source,terminal]
+----
+$ ./openshift-mirror-registry install \
+-k ~/.ssh/my_ssh_key --quayHostname some.remote.host.com
+----
+
+. Once installed, the Red Hat Quay console is accessible at `https://<quayhostname>:8443`. Use the user name and password generated during installation to log into the registry by running the following command:
++
+[source,terminal]
+----
+$ podman login -u init -p <password> --tls-verify=false quay:8443
+----
+
+. After logging in, you can tag, push, and pull your images. For example:
++
+[source,terminal]
+----
+$ podman tag docker.io/library/busybox:latest quay:8080/init/busybox:latest
+----
++
+[source,terminal]
+----
+$ podman push quay:8443/init/busybox:latest --tls-verify=false
+----
++
+[source,terminal]
+----
+$ podman pull quay:8443/init/busybox:latest --tls-verify=false
+----
+
+. You can uninstall Red Hat Quay from your local host by running the following command:
++
+[source,terminal]
+----
+$ sudo ./openshift-mirror-registry uninstall -v
+----
++
+[NOTE]
+====
+Uninstalling your registry will delete the `/etc/quay-install` directory and disable all systemd services set up by Quay.
```

Comment:
Can this be updated to include reference for deleting a custom `quayRoot`? If a user installs with a `quayRoot`, say, `/home/ec2-user/quay`, in order to delete that they will need to pass in `--quayRoot /home/ec2-user/quay` to the uninstall command. This is caught in https://issues.redhat.com/browse/PROJQUAY-3009

Reply:
Can we also include a note that all uninstalls will prompt the user before deleting `quayRoot`? This prompt can be skipped by passing `--autoApprove` to the uninstall command.